zk.go

```diff
@@ -21,12 +25,18 @@ type zkConsumerConfig struct {
 }
 
 // NewZK creates a new connection instance
-func NewZK(servers []string, recvTimeout time.Duration) (*ZK, error) {
+func NewZK(servers []string, chrootPath string, recvTimeout time.Duration) (*ZK, error) {
```

Comment:
I'd love to see a few more tests for `NewZK` too, e.g. `It("should initialize with path prefix", func() { ... })` with a few prefix examples.